.coafile

```diff
@@ -0,0 +1,20 @@
+[all]
```

Comment:
have a look at https://github.com/thoth-station/user-api/blob/master/.coafile I think it is a better variant

Reply:
+1, updated.